src/rdkafka_conf.h

```diff
@@ -527,6 +528,9 @@ struct rd_kafka_topic_conf_s {
 				void *msg_opaque);
         char   *partitioner_str;
 
+        int random_partitioner; /**< 1 - sticky
```

Comment:
This naming is a bit ambigious, if random_partitioner is set to true it is not random but sticky?